main.go

```diff
@@ -150,10 +154,6 @@ func initSelfNodeRemediationManager(mgr manager.Manager, enableHTTP2 bool) {
 
 	configureWebhookServer(mgr, enableHTTP2)
 
-	if err := utils.InitOutOfServiceTaintSupportedFlag(mgr.GetConfig()); err != nil {
-		setupLog.Error(err, "unable to verify out of service taint support. out of service taint isn't supported")
```

Comment:
NIT: Sometimes we refer to the taint as `out of service` with spaces and sometimes without spaces `OutOfService` in comments and prints. I don't have a strong preference for these two, but I see that `out-of-service` is used in the [official Kubernetes documentation](https://kubernetes.io/docs/reference/labels-annotations-taints/#node-kubernetes-io-out-of-service). So if we want to change it or to align ourself, thus I would suggest using `out-of-service`.
We can probably do this change under [common](https://github.com/medik8s/common/tree/main/pkg) and then SNR and FAR can use it after the [FAR out-of-service PR](https://github.com/medik8s/fence-agents-remediation/pull/92) is merged.

I also think that this kind of PR will be meaningful for FAR eventually.

Reply:
Not sure if I'd go as far as using common for text in log messages, but I agree that sticking to official docs with 'out-of-taint' is a good idea.
BTW this comment was misplaced - it's on removed line, but still relevant elsewhere 🙂

Reply:
>BTW this comment was misplaced - it's on removed line, but still relevant elsewhere 

Correct, and there few more places that we print `outofservice` if you want to fix:

- [https://github.com/medik8s/self-node-remediation/pull/160/files#diff-dd6f1328b9432817228e47edffdc3fe1e4570bfa2e39099223c05131e89e9068R871 and ](https://github.com/medik8s/self-node-remediation/pull/160/files#diff-dd6f1328b9432817228e47edffdc3fe1e4570bfa2e39099223c05131e89e9068R870-R871)
- https://github.com/medik8s/self-node-remediation/pull/160/files#diff-dd6f1328b9432817228e47edffdc3fe1e4570bfa2e39099223c05131e89e9068R848-R849

Reply:
Thanks, good catch ! - fixed